### ISSUE TITLE
[Woo POS] Improve Voice Over support

### ISSUE THREAD

**Issue by staskus** [woocommerce/woocommerce-ios#13517]
Review the flows using Voice Over and make improvements:

- Hide non-descriptive images from accessibility
- Group items, especially totals lines
- Make sure prices are read correctly with currency
- Provide accessibility descriptions
- Make sure accessibility elements are not visible behind the popup

### From initial accessibility review
- [x] Section heading accessibility traits (Products, Cart, Totals, Reader message titles (by styling), error message titles)
- [x] Make VStacks group children where required (so it's clear when you're in the cart or item list)
- [x] Hide decorative images on errors
- [x] Hide decorative images on empty states
- [x] Make it possible to select the `...` and `Connect your reader` buttons before adding to the cart
- [x] Remove from cart `x` button announced as "Close" – provide a sensible name
- [x] #13668
- [x] Hide Card Reader Connection flow images
- [x] Select `Check out` button after cart
- [x] Hide Card payment images
- [x] Hide "Circle" in card reader connection view

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by joshheald** [comment]
Done with #13668
